*** Starting uWSGI 2.0.17 (64bit) on [Thu May 17 13:18:40 2018] ***
compiled with version: 5.4.0 20160609 on 17 May 2018 13:03:13
os: Linux-4.4.0-1052-aws #61-Ubuntu SMP Mon Feb 12 23:05:58 UTC 2018
nodename: ip-172-31-33-186
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/filmspace_20180517/website
writing pidfile to uwsgi.pid
open("uwsgi.pid"): Permission denied [core/utils.c line 3605]
*** Starting uWSGI 2.0.17 (64bit) on [Thu May 17 15:52:25 2018] ***
compiled with version: 5.4.0 20160609 on 17 May 2018 13:03:13
os: Linux-4.4.0-1052-aws #61-Ubuntu SMP Mon Feb 12 23:05:58 UTC 2018
nodename: ip-172-31-33-186
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/filmspace_20180517/website
detected binary path: /home/ubuntu/venv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/filmspace_20180517/website
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 3900
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address :8077 fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 23 2017, 16:37:01)  [GCC 5.4.0 20160609]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x11afe70
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 291616 bytes (284 KB) for 4 cores
*** Operational MODE: preforking ***
Traceback (most recent call last):
  File "./wsgi.py", line 4, in <module>
    reload(sys)
NameError: name 'reload' is not defined
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (pid: 29195, cores: 1)
spawned uWSGI worker 2 (pid: 29196, cores: 1)
spawned uWSGI worker 3 (pid: 29197, cores: 1)
spawned uWSGI worker 4 (pid: 29198, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 29196|app: -1|req: -1/1] 27.38.250.247 () {42 vars in 706 bytes} [Thu May 17 15:52:50 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 29195|app: -1|req: -1/2] 27.38.250.247 () {44 vars in 710 bytes} [Thu May 17 15:52:51 2018] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
*** Starting uWSGI 2.0.17 (64bit) on [Thu May 17 15:56:52 2018] ***
compiled with version: 5.4.0 20160609 on 17 May 2018 13:03:13
os: Linux-4.4.0-1052-aws #61-Ubuntu SMP Mon Feb 12 23:05:58 UTC 2018
nodename: ip-172-31-33-186
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/filmspace_20180517/website
detected binary path: /home/ubuntu/venv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/filmspace_20180517/website
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 3900
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
probably another instance of uWSGI is running on the same address (:8077).
bind(): Address already in use [core/socket.c line 769]
*** Starting uWSGI 2.0.17 (64bit) on [Thu May 17 15:59:54 2018] ***
compiled with version: 5.4.0 20160609 on 17 May 2018 13:03:13
os: Linux-4.4.0-1052-aws #61-Ubuntu SMP Mon Feb 12 23:05:58 UTC 2018
nodename: ip-172-31-33-186
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /home/filmspace_20180517/website
detected binary path: /home/ubuntu/venv/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/filmspace_20180517/website
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 3900
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
probably another instance of uWSGI is running on the same address (:8077).
bind(): Address already in use [core/socket.c line 769]
--- no python application found, check your startup logs for errors ---
[pid: 29196|app: -1|req: -1/3] 27.38.250.247 () {40 vars in 675 bytes} [Thu May 17 16:00:21 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 29195|app: -1|req: -1/4] 27.38.250.247 () {40 vars in 657 bytes} [Thu May 17 16:00:22 2018] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 29196|app: -1|req: -1/5] 80.88.90.131 () {34 vars in 507 bytes} [Thu May 17 16:16:53 2018] HEAD /phpmyadmin/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 29195|app: -1|req: -1/6] 80.88.90.131 () {34 vars in 493 bytes} [Thu May 17 16:16:53 2018] HEAD /PMA/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 29196|app: -1|req: -1/7] 80.88.90.131 () {34 vars in 501 bytes} [Thu May 17 16:16:54 2018] HEAD /dbadmin/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 29197|app: -1|req: -1/8] 80.88.90.131 () {34 vars in 493 bytes} [Thu May 17 16:16:54 2018] HEAD /pma/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 29198|app: -1|req: -1/9] 80.88.90.131 () {34 vars in 491 bytes} [Thu May 17 16:16:55 2018] HEAD /db/ => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
